README.md

```diff
@@ -1,11 +1,17 @@
 # elastic-package
 
-The elastic-package is a command line tool, written in Go, used for developing Elastic Integrations.
+The elastic-package is a command line tool, written in Go, used for developing Elastic packages.
+
+*For experimental use only*
 
 ## Features
 
 TODO
 
+## Supported packages
```

Comment:
Let's do "package types" so the `type` key will sound more familiar if an user peeks into a manifest

Reply:
Agree.

Changed to "Supported package types".